custom/enikshay/nikshay_datamigration/models.py

```diff
@@ -285,8 +285,8 @@ class Outcome(models.Model):
     )
     InitiatedDate = models.CharField(max_length=255, null=True)  # dates, None, and NULL
     userName = models.CharField(max_length=255)
-    loginDate = models.DateTimeField()
-    OutcomeDate1 = models.CharField(max_length=255)  # datetimes and NULL
+    # loginDate = models.DateTimeField()
+    # OutcomeDate1 = models.CharField(max_length=255)  # datetimes and NULL
```

Comment:
why did you do this this way as opposed to adding a migration to remove them? just feels like manually changing migration files might get us into trouble.